.travis.yml

```diff
@@ -186,8 +180,8 @@ matrix:
     # JDK9 build
     - jdk: oraclejdk9
       env:
-        - DESC="build with JDK9"
-        - CMD="mvn -e package -Passembly"
+        - DESC="JDK9 site, assembly"
+        - CMD="mvn -e site -Dlinkcheck.skip=true && mvn -e package -Passembly"
```

Comment:
please move it to separate commit(with reference jdk9 issue) or to another PR.
cobertura --> jacoco is unrelated to jdk8 --> jdk9 . Even they a bit bound.

Reply:
done. Reverted to original `CMD="mvn -e package -Passembly"`